pkg/convert/tf.go

```diff
@@ -2869,10 +2960,74 @@ func translateRemoteModule(
 		destinationRoot, destinationDirectory,
 		info,
 		requiredProviders,
+		sandboxedModules,
 		/*topLevelModule*/ false,
 	)
 }
 
+func sanboxedAnnotation(trivia string) (string, bool) {
+	lines := strings.Split(trivia, "\n")
+	for _, line := range lines {
+		parts := strings.Split(strings.ReplaceAll(line, "//", ""), " ")
+		partsLength := len(parts)
+		for i, part := range parts {
+			if part == sandboxModuleAnnotation && i < partsLength-1 {
+				packageName := parts[i+1]
+				return packageName, true
+			}
+		}
+	}
+
+	return "", false
+}
+
+func moduleFromRemoteRegistry(moduleCall *configs.ModuleCall) (addrs.ModuleSourceRegistry, bool) {
```

Comment:
Why does the code needs to branch on this bool returned here? What's different for non-remote modules?

Reply:
We don't handle non-remote modules i.e. local with this implementation

Reply:
Why not? That is kind of essential. That is our most important use case for enterprises trying to transition legacy modules maintained locally. What would it take?